rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -27,6 +27,11 @@
 logger = logging.getLogger('rosdoc2')
 
 
+def esc_backslash(path):
+    """Escape backslashes to support Windows paths in strings."""
+    return path.replace('\\', '\\\\') if path else path
```

Comment:
Couldn't instantiating `pathlib.Path` objects handle this?

Reply:
I like that idea but think that this is going into the config of the next stage where it is then needed to be escaped. 

I'd suggest considering switching to this as an enhancement for the future.

Reply:
I've opened a ticket to follow up on this https://github.com/ros-infrastructure/rosdoc2/issues/50